### PR TITLE
Add mcu support to the carousel example

### PR DESCRIPTION
**Pull request by FloVanGH**
Now it is possible to run the carousel example on the boards supported by the `mcu-board-support` crate. It contains also some visual improvements to looks nicer on smaller screens.

### CODE REVIEW

**Review by ogoffart — COMMENTED** [review]
Why move the assets since they are used in only one demo anyway?

**Comment by FloVanGH** [comment]
I moved it because of the extra mcu example. You're right it's no more necessary.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -57,12 +57,12 @@ jobs:
       uses: actions-rs/cargo@v1
       with:
           command: build
-          args: --verbose --all-features --workspace ${{ matrix.extra_args }} --exclude test-driver-cpp --exclude mcu-board-support --exclude printerdemo_mcu  # mcu backend requires nightly
+          args: --verbose --all-features --workspace ${{ matrix.extra_args }} --exclude test-driver-cpp --exclude mcu-board-support --exclude printerdemo_mcu --exclude carousel  # mcu backend requires nightly
```
[the remark this answers is not in the document]
Comment (FloVanGH) [reply .github/workflows/ci.yaml:60]: Yes because of all-features

**Review by ogoffart — APPROVED** [review]
(please squash)

**Pull request merged**